pallets/api/src/fungibles/tests.rs

```diff
@@ -1,11 +1,20 @@
 use crate::mock::*;
 use frame_support::{
-	assert_ok,
+	assert_noop, assert_ok,
 	traits::fungibles::{approvals::Inspect, metadata::Inspect as MetadataInspect},
 };
+use sp_runtime::{DispatchError, ModuleError};
 
 const ASSET: u32 = 42;
 
+fn get_dispatch_error(index: u8, error_index: u8, error_message: &'static str) -> DispatchError {
```

Comment:
Can be removed, we don't emit errors in this pallet and we only want to test the functionality we add to the pallet, not the dispatchables itself.